tutorecommerce/templates/ecommerce/apps/ecommerce/settings/development.py

```diff
@@ -2,11 +2,13 @@
 
 {% include "ecommerce/apps/ecommerce/settings/partials/common.py" %}
 
-CORS_ORIGIN_WHITELIST = list(CORS_ORIGIN_WHITELIST) + [
-    "http://{{ MFE_HOST }}:{{ ECOMMERCE_MFE_APP['port'] }}",
-    "http://{{ MFE_HOST }}:{{ ECOMMERCE_PAYMENT_MFE_APP['port'] }}",
-]
-CSRF_TRUSTED_ORIGINS = ["{{ MFE_HOST }}:{{ ECOMMERCE_MFE_APP['port'] }}"]
+{% if ECOMMERCE_MFE_APP and ECOMMERCE_PAYMENT_MFE_APP %}
+    CORS_ORIGIN_WHITELIST = list(CORS_ORIGIN_WHITELIST) + [
+        "http://{{ MFE_HOST }}:{{ ECOMMERCE_MFE_APP['port'] }}",
+        "http://{{ MFE_HOST }}:{{ ECOMMERCE_PAYMENT_MFE_APP['port'] }}",
+    ]
+    CSRF_TRUSTED_ORIGINS = ["{{ MFE_HOST }}:{{ ECOMMERCE_MFE_APP['port'] }}"]
```

Comment:
Identation issues. @Squirrel18 
```suggestion
CORS_ORIGIN_WHITELIST = list(CORS_ORIGIN_WHITELIST) + [
    "http://{{ MFE_HOST }}:{{ ECOMMERCE_MFE_APP['port'] }}",
    "http://{{ MFE_HOST }}:{{ ECOMMERCE_PAYMENT_MFE_APP['port'] }}",
]
CSRF_TRUSTED_ORIGINS = ["{{ MFE_HOST }}:{{ ECOMMERCE_MFE_APP['port'] }}"]
```